src/catkin_pkg/changelog_generator_vcs.py

```diff
@@ -128,7 +128,7 @@ def _truncate_timestamps(self, tags):
 
 class GitClient(VcsClientBase):
 
-    type = 'git'
+    type = 'git'  # flake8: noqa
```

Comment:
Please ignore only the specific code instead of a "wildcard", e.g. `# noqa: A003`. Otherwise additional warnings might be masked.

Reply:
Fixed.